src/components/thumbnail/tab-panel-documents-section.sass

```diff
@@ -30,7 +30,7 @@ $list-item-scale: 0.11
 
       .scaled-list-item-container
         border-radius: 5px
-        border: solid 1.5px $charcoal-light-1
+        border: solid 2px $charcoal-light-2
```

Comment:
I'm confused.  Is there a new spec?  I'm seeing that this should be 1.5px and charcoal-light-1.  Where are you seeing these changes?
https://app.zeplin.io/project/5d62a554d64a9e02dcad80de/screen/5f7b52e62e06e6696a735f06